dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/user/DefaultUserService.java

```diff
@@ -700,6 +700,12 @@ public boolean credentialsNonExpired( UserCredentials credentials )
         return months < credentialsExpires;
     }
 
+    @Override
+    public boolean isAccountExpired( UserCredentials credentials )
+    {
+        return !credentials.isAccountNonExpired();
```

Comment:
This can mildly problematic in the case where an object object is looked up outside the transactional context, then passed into a transactional service method where a lazy-loaded collection is accessed on the object (lazy load exception). I think it is fine for now though.

Reply:
yes, this method is not annotated `@Transactional` intentional.